lib/src/sip_ua_helper.dart

```diff
@@ -412,9 +412,9 @@ class Call {
     refer.on(EventReferFailed(), (EventReferFailed data) {});
   }
 
-  void hangup() {
+  void hangup(Map<String, dynamic> options) {
```

Comment:
I think there needs to be a preset parameter here, so that it will not break other projects that rely on this api after the update.

look like this.
`void hangup([Map<String, Object> options]) {`